fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Request.kt

```diff
@@ -140,6 +140,15 @@ class Request : Fuel.RequestConvertible {
         return this
     }
 
+    fun source(source: (Request, URL) -> File): Request {
```

Comment:
Can we implement `source` call `sources` internally so we can save some duplications?